src/VisualStudio/CSharp/Impl/Options/CSharpSettingsManagerOptionSerializer.cs

```diff
@@ -274,7 +296,42 @@ public override bool TryPersist(OptionKey optionKey, object value)
                 }
             }
 
+            // code style: use var options.
+            if (optionKey.Option == CSharpCodeStyleOptions.UseImplicitTypeForIntrinsicTypes)
+            {
+                return PersistUseVarOption(Style_UseVarForIntrinsicTypes, value);
+            }
+            else if (optionKey.Option == CSharpCodeStyleOptions.UseImplicitTypeWhereApparent)
+            {
+                return PersistUseVarOption(Style_UseVarWhenTypeIsApparent, value);
```

Comment:
yeah, my bad. I only touched user facing strings and variables. which is why the mismatch. I'll fix.